sdk/keyvault/azure-keyvault-keys/samples/list_operations.py

```diff
@@ -1,10 +1,10 @@
 import time
 import os
-from azure.security.keyvault import KeyClient
+from azure.keyvault.keys import KeyClient
 from azure.identity import DefaultAzureCredential
 from azure.core.exceptions import HttpResponseError
 
- # ----------------------------------------------------------------------------------------------------------
+# ----------------------------------------------------------------------------------------------------------
 # Prerequistes -
 #
 # 1. An Azure Key Vault-
```

Comment:
Need to remove `security` from the package name on point `2` and update the name also to reflect the  corresponding package name in all the samples files.
>  2. Microsoft Azure Key Vault PyPI package -
> https://pypi.python.org/pypi/azure-security-keyvault/